association_backend/association/stripe/templates/payment_ok.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```

Comment:
reminder to remove all of this

Reply:
stripe/templates/* will be removed
Without these it was impossible for me to complete the flow

If you want I can remove them